org/wildfly/transactions/capability.adoc

```diff
@@ -0,0 +1,29 @@
+NAME: org.wildfly.transactions
+
+DESCRIPTION: Provides the transaction manager.
+
+REGISTERED BY:
+  
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: false
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: true
+
+SERVICE PROVIDED:
+
+  CLASS: org.jboss.as.txn.service.TransactionManagerService
```

Comment:
@zhfeng I am wondering how https://github.com/wildfly/wildfly/blob/master/transactions/src/main/java/org/jboss/as/txn/service/UserTransactionService.java#L40 might also factor into describing the transaction capabilities?

Reply:
I am not sure the two class names can be added here. 
```
CLASS: org.jboss.as.txn.service.TransactionManagerService, org.jboss.as.txn.service.UserTransactionService
```

Reply:
@jmesnil would you have some guidance on that please?